package HTTP_Record;

message http_message {
    required bytes first_line = 1;
    repeated bytes headers = 2;
    optional bytes body = 3;
}

message reqrespair {
    required http_message req = 1;
    required http_message res = 2;
}
